setup.py

```diff
@@ -202,11 +202,13 @@ def __str__(self):
         libraries=['m',PROTEUS_LAPACK_LIB,
                    PROTEUS_BLAS_LIB]
     ),
-    Extension("mprans.cRANS3PF",['proteus/mprans/cRANS3PF.pyx'],
-              depends=['proteus/mprans/RANS3PF.h','proteus/mprans/RANS3PF2D.h', 'proteus/ModelFactory.h', 'proteus/CompKernel.h'],
-              language='c++',
-              extra_compile_args=PROTEUS_OPT+["-std=c++14","-mavx"],
-              include_dirs=get_xtensor_include()),
+    Extension(
+        'mprans.RANS3PF',
```

Comment:
mprans.RANS3PF ==> mprans.cRANS3PF

Reply:
I'm gonna make a global review of these names, it looks like I missed some naming inconsistency between the setup.py, the XXX.py files and the XXX.cpp iles.